openapi-core/src/main/java/io/ballerina/openapi/core/generators/common/OASModifier.java

```diff
@@ -621,7 +408,7 @@ public static Map<String, String> collectParameterNames(List<Parameter> paramete
     }
 
     public static String replaceContentInBraces(String input, String expectedValue) {
-        String regex = "\\{([^}]*)\\}";
+        String regex = "\\{([^}]*)}";
```

Comment:
Lets mark this as a constant as well.

Reply:
Fixed via [21628fe](https://github.com/ballerina-platform/openapi-tools/pull/1766/commits/21628fe6c77a0f54d70984c8e48d6718612b1319)